test/e2e/framework.go

```diff
@@ -64,6 +64,7 @@ type TestOptions struct {
 	operatorConfigPath  string
 	logsExportOnSuccess bool
 	withIPPool          bool
```

Comment:
I think can remove this flag `withIPPool`, @zhengxiexie  since we don't have ippool test.

Reply:
Thanks for reviewing. Deleted.